Modules/System Tests/Azure Blob Services API/src/ABSTestLibrary.Codeunit.al

```diff
@@ -152,6 +152,129 @@ codeunit 132921 "ABS Test Library"
         exit(Document);
     end;
 
+    procedure GetServiceResponseBlobWithHierarchicalName() ResponseText: Text;
+    begin
+        ResponseText := '<?xml version="1.0" encoding="utf-8"?>';
```

Comment:
We should use StringBuilder instead of string concatenation for Creating Test Responses for performance reasons

Reply:
Changed concatenation to TextBuilder. Although, to be honest, I would not bother about performance too much in this case. Performance difference between the two versions will be in the order of magnitude of nanoseconds (considering that no fancy compiler optimisation happens behind the scene, which can eliminate any difference). Not a performance bottleneck, I'd say.